package.json

```diff
@@ -17,6 +17,7 @@
     "preview": "vite preview"
   },
   "devDependencies": {
+    "@blocknote/mantine": "^0.39.0",
```

Comment:
So this is a dev dependency now? Don't we need that as a regular dependency or peerDependency?

Reply:
I mentioned yesterday that it's possible to setup BlockNote with Mantine or [AriaKit](https://www.blocknotejs.org/docs/getting-started/ariakit) (and there's yet another option).

So my thought process was to make the component library an internal dependency for op-blocknote-extensions, and only leave as *peerDependencies* the other two blocknote ones. This should allow whoever consumes this library to use any frontend toolkit they wish

Reply:
But I assume if we write our macros with that frontend-kit (whichever we decide on), then probably everyone who wants to use this plugin also needs that frontend-kit so that our blocks etc. render correctly? Otherwise we would have to developt and test against multiple frontend-kits, wouldn't we? :sweat_smile: